src/test/resources/epub3/mediaoverlays-document.feature

```diff
@@ -10,5 +10,82 @@ Feature: EPUB 3 Media Overlays Document
         are defined in the `mediaoverlays.feature` feature file.
 
   Background: 
-    Given EPUB test files located at '/epub3/files/epub/'
-    And EPUBCheck with default settings
+    Given EPUB test files located at '/epub3/files/mediaoverlays-document/'
+    And EPUBCheck configured to check a Media Overlays Document
+
+
+  Scenario: Valid SMIL Document with hh:mm:ss.milli format
+    When checking document 'valid-timestamp-format.smil'
+    Then no errors or warnings are reported
+
+
+  Scenario: Valid SMIL Document with XX min format
+    When checking document 'valid-word-format.smil'
+    Then no errors or warnings are reported
+
+  Scenario: Valid SMIL Document with a head section and meta data
+    When checking document 'valid-head-metadata.smil'
+    Then no errors or warnings are reported
+
+  Scenario: Valid SMIL Document with epub:Type value of "aside"
+    When checking document 'valid-epubtype-aside.smil'
+    Then no errors or warnings are reported
+
+  Scenario: Valid SMIL Document with additional prefixes declared
+    When checking document 'valid-prefixes.smil'
+    Then no errors or warnings are reported
+
+  Scenario: SMIL Document with deprecated head syntax
+    When checking document 'incorrect-meta-data.smil'
+    Then error RSC-005 is reported
+    And the message contains 'element "meta" not allowed here'
+    And no other errors or warnings are reported
+
+  Scenario: SMIL Document with duplicate siblings
+    When checking document 'duplicate-siblings.smil'
+    Then error RSC-005 is reported
+    And the message contains 'element "text" not allowed here'
+    And no other errors or warnings are reported
+
+  Scenario: SMIL Document with `seq` nested in a `par` element.
+    When checking document 'seq-nested-in-par.smil'
+    Then the following errors are reported
+        | RSC-005 | element "seq" not allowed here |
+        | RSC-005 | element "seq" missing required attribute "epub:textref" |
+    And no other errors or warnings are reported
+
+  Scenario: SMIL Document with `audio` and `text` elements in a seq element
+    When checking document 'audio-nested-in-seq.smil'
+    Then the following errors are reported
+        | RSC-005 | element "audio" not allowed here |
+        | RSC-005 | element "text" not allowed here |
+    And no other errors or warnings are reported
+
+  Scenario: SMIL Document with invalid clip-begin and clip-end formats
+    When checking document 'incorrect-clip-formats.smil'
+    Then the following errors are reported
+        | RSC-005 | value of attribute "clipBegin" is invalid |
+        | RSC-005 | value of attribute "clipEnd" is invalid |
+        | RSC-005 | value of attribute "clipBegin" is invalid |
+        | RSC-005 | value of attribute "clipEnd" is invalid |
+        | RSC-005 | value of attribute "clipBegin" is invalid |
+        | RSC-005 | value of attribute "clipEnd" is invalid |
+        | RSC-005 | value of attribute "clipBegin" is invalid |
```

Comment:
to reduce the verbosity of this scenario, we could just count the `RSC-005` errors here:
```
Then error RSC-005 is reported 7 times
```

If we really wanted to check the message content, we could just check the last message, like:

```
And the last message contains 'value of attribute "clipBegin" is invalid'
```

(currently "last" above is not allowed, but I can amend the step definition if needed).